db/migrate/20170217085547_add_type_to_cloud_object_store_container.rb

```diff
@@ -0,0 +1,17 @@
+class AddTypeToCloudObjectStoreContainer < ActiveRecord::Migration[5.0]
+  class CloudObjectStoreContainer < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  def up
+    add_column :cloud_object_store_containers, :type, :string
+    add_index :cloud_object_store_containers, [:type, :ems_id]
```

Comment:
I don't understand the compound index here.  type, and ems_id independently make since, but I don't understand them together.